test/common/common/logger_test.cc

```diff
@@ -560,6 +564,24 @@ TEST(TaggedLogTest, TestConnLog) {
   object.logMessageWithConnection();
 }
 
+TEST(TaggedLogTest, TestConnEventLog) {
```

Comment:
lg, could you please also test the JSON case?

Reply:
Added json test anyway to make the future change consistent. There is no customized `Tags` for this marco right now.